extension/src/paymentHandler/payment-handler.js

```diff
@@ -122,21 +119,21 @@ function _validatePaymentRequest(paymentObject, authToken) {
     if (paymentValidator.hasErrors())
       return {
         success: false,
-        data: paymentValidator.buildCtpErrorResponse(),
+        errors: paymentValidator.buildCtpErrorResponse()?.errors,
```

Comment:
Do you need the null check here again? Seems like you check already `paymentValidator.hasErrors()`?

Reply:
you're right. `errors` inside `paymentValidator` is always non-null
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/64058c6cac27e381313dc82dee9149fd8f7a3d80

Reply:
You have this check in multiple places.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/b34bccc36c56fe0f323e5bd457a8ae5cc3c8aaef